.teamcity/settings.kts

```diff
@@ -74,4 +78,171 @@ object Chocolatey : BuildType({
             }
         }
     }
+
+    requirements {
+        doesNotExist("docker.server.version")
+    }
+})
+
+object ChocolateyDockerWin : BuildType({
+    id = AbsoluteId("ChocolateyDockerWin")
+    name = "Docker (Windows)"
+
+    params {
+        // TeamCity has suggested "${Chocolatey.depParamRefs.buildNumber}"
+        param("env.CHOCOLATEY_VERSION", "%dep.Chocolatey.build.number%")
+        param("env.vcsroot.branch", "%vcsroot.branch%")
+        param("env.Git_Branch", "%teamcity.build.vcs.branch.Chocolatey_ChocolateyVcsRoot%")
+        param("teamcity.git.fetchAllHeads", "true")
+    }
+
+    vcs {
+        root(DslContext.settingsRoot)
+    }
+
+    steps {
+        script {
+            name = "Call Cake"
+            scriptContent = "call build.official.bat --verbosity=diagnostic --target=Docker"
+        }
+
+        // powerShell {
```

Comment:
Is this section still needed?

Reply:
I was going to ask you if you wanted this in or no. I don't think it adds terribly much, though we could do a more active check, storing `choco version` and comparing it to an expected value?

Reply:
Removed.